### PR TITLE
Ethers configuration + page layout

### PR DESCRIPTION
**Pull request by RcleydsonR**
To full use the application, it is needed to run [smart contract](https://github.com/doiim/p2pix-smart-contracts) in localhost.

### CODE REVIEW

File: src/utils/ethers.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+const makeTransaction = async () => {
+  const etherStore = useEtherStore();
+  const provider = getProvider();
+  if (!provider) return;
+
+  const signer = provider.getSigner();
+  const contract = new ethers.Contract(addresses.token, mockToken.abi, signer);
+
+  const fixedAccount1Address = "0x70997970C51812dc3A010C7d01b50e0d17dc79C8";
```
Comment (Ronyell) [inline src/utils/ethers.ts:53]: Add a comment with TODO to change this variable when this accout will pass for this function (maybe define default value for this variable in function signature is better)

File: src/utils/ethers.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  const tx = await contract.transfer(
+    fixedAccount1Address,
+    ethers.utils.parseEther("100.0")
```
Comment (Ronyell) [inline src/utils/ethers.ts:56]: This number "100.00" the same.  Define default value for this variable in function signature.

File: src/utils/ethers.ts (hunk 1 above)
Comment (RcleydsonR) [reply src/utils/ethers.ts]: Thx. Fixed.

File: src/utils/ethers.ts (hunk 2 above)
Comment (RcleydsonR) [reply src/utils/ethers.ts]: Thx. Fixed.

**Review by Ronyell — APPROVED** [review]

**Pull request merged**